api/v1alpha1/accesslogging_types.go

```diff
@@ -20,6 +20,7 @@ type ProxyAccessLogSetting struct {
 	// Format defines the format of accesslog.
 	Format ProxyAccessLogFormat `json:"format"`
 	// Matches defines the match conditions for accesslog in CEL expression.
+	// An accesslog will be emitted only when one or more match conditions are evaluated to true.
```

Comment:
curious if we can use CEL to validate the CEL 🙃  ? Or will it be hard keeping track of upstream OSS attributes?

Reply:
TBH, I don't know, mostly can't.